### PR TITLE
Amp animation e2e test

### PR DESCRIPTION
**Pull request by powerivq**
Similar purpose: We want to use e2e to ensure amp-animation does not broke horribly within amp4ads (inabox)

### CODE REVIEW

File: extensions/amp-animation/0.1/test-e2e/test-amp-animation.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-animation',
+  {
+    testUrl:
+      'http://localhost:8000/test/fixtures/e2e/amp-animation/simple.html',
+    environments: 'ampdoc-amp4ads-preset',
```
Comment (kristoferbaxter) [inline extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: Ignorance here, but does this mean the `e2e` validates the `a4a` case only?

File: extensions/amp-animation/0.1/test-e2e/test-amp-animation.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  env => {
+    let controller;
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    it('simple animation', async () => {
```
Comment (kristoferbaxter) [inline extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: Is it enough coverage to test a simple animation alone or should more complex tests be added as well?

Comment (cathyxz) [inline extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: Nit: test naming should be descriptive about what behavior the test asserts for.

File: extensions/amp-animation/0.1/test-e2e/test-amp-animation.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      const cancelBtn = await controller.findElement('#cancelBtn');
+      await controller.click(cancelBtn);
+
+      const elem = await controller.findElement('#image');
+      await expect(controller.getElementCssValue(elem, 'opacity')).to.be.equal(
```
Comment (lannka) [inline extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: nit: `to.equal`

File: extensions/amp-animation/0.1/test-e2e/test-amp-animation.js (hunk 1 above)
Comment (lannka) [reply extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: +1, any difficulty here to enable the test for all envs?

Comment (lannka) [reply extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: oh, actually this preset covers all envs, including ampdoc & a4a.

File: extensions/amp-animation/0.1/test-e2e/test-amp-animation.js (hunk 2 above)
Comment (powerivq) [reply extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: Thanks!

Comment (powerivq) [reply extensions/amp-animation/0.1/test-e2e/test-amp-animation.js]: Yeah, I think we will.

**Comment by rsimha** [comment]
@powerivq These tests consistently fail on `master`: https://travis-ci.org/ampproject/amphtml/jobs/597756202#L2054

Can you investigate?

/cc @jridgewell who is build cop

**Pull request merged**